server/src/main/java/org/elasticsearch/rest/action/RestActions.java

```diff
@@ -268,7 +260,7 @@ public RestResponse buildResponse(NodesResponse response, XContentBuilder builde
     /**
      * Parses a top level query including the query element that wraps it
      */
-    private static QueryBuilder parseTopLevelQueryBuilder(String fieldName, XContentParser parser) {
+    public static QueryBuilder getQueryContent(XContentParser parser) {
```

Comment:
Does this rename make sense or is it left over from a previous iteration?

Reply:
I think it makes sense as it allows removing one method without affecting consumers.